### PR TITLE
All request digital for available reserve items

### PR DESCRIPTION
**Pull request by JPrevost**
- Status 24 items (high use, usually on resevre) not eligible for Request Item, but need to be eligible for Request Digital Copy

https://mitlibraries.atlassian.net/browse/DI-932

- Also removes the note to ask at desk for reserve items

#### Requires Database Migrations?
NO

#### Includes new or updated dependencies?
NO

### CODE REVIEW

File: app/models/concerns/processing_labeler.rb [hunk 1]
```diff
@@ -90,7 +90,6 @@ def in_transit?(status)
   # expect. This string is based on the item status in Aleph, but we edit it
   # for readability.
   def processing_label(status, z30_item_process, reserve)
-    return 'On reserve (see desk)' if reserve
```
Comment (matt-bernhardt) [inline app/models/concerns/processing_labeler.rb:93]: Are we using the reserve parameter anywhere else in this method (or related code)? Or can we remove it from the method and the calling code entirely?

Comment (JPrevost) [reply app/models/concerns/processing_labeler.rb:93]: Good question.

Yes we still use it for some of the general button logic (some of which is deactivated as this time, but will come back one day with some ENV changes when needed).

Comment (JPrevost) [reply app/models/concerns/processing_labeler.rb:93]: Oh wait, I think I misunderstood your question. We indeed don't use it in this method and can likely clean this up. Let me check (my response was based on we still care about reserves elsewhere but you clearly asked about this method and I answered wrongly... sorry). Great catch! Will check and likely clean this up.

Comment (JPrevost) [reply app/models/concerns/processing_labeler.rb:93]: @matt-bernhardt I've made the suggested change. Thanks.

Comment (matt-bernhardt) [reply app/models/concerns/processing_labeler.rb:93]: Yep - that's what I was thinking of. Thanks! Thumbing next

**Review by matt-bernhardt — APPROVED** [review]
This looks good - :shipit:

**Pull request merged**